src/operators/OperatorDetails.jsx

```diff
@@ -16,19 +16,21 @@ import {
   TableColumn,
   TableHeader,
   TableRow,
-  Tabs
+  Tabs,
+  Tooltip
 } from '@nextui-org/react';
+import { useEffect, useMemo, useState } from 'react';
 import { useLocation, useNavigate, useParams } from 'react-router-dom';
 import CopyButton from '../shared/CopyButton';
 import ErrorMessage from '../shared/ErrorMessage';
-import { formatETH } from '../shared/formatters';
-import OperatorLSTPieChart from './charts/OperatorLSTPieChart';
+import { formatEther } from 'ethers';
 import OperatorRestakersLineChart from './charts/OperatorRestakersLineChart';
 import OperatorTVLLineChart from './charts/OperatorTVLLineChart';
 import { ParentSize } from '@visx/responsive';
 import ThirdPartyLogo from '../shared/ThirdPartyLogo';
+import { tooltip } from '../shared/slots';
 import { truncateAddress } from '../shared/helpers';
-import { useEffect } from 'react';
+import TVLTabTreemap from './charts/TVLTabTreemap';
```

Comment:
As the operator treemap is identical to the AVS one, let's simplify it and reuse the AVS one. Later, if anything changes somewhere, we can tune it for that.

```suggestion
import TVLTabTreemap from '../avs/charts/TVLTabTreemap';
```